.github/workflows/release.yml

```
@@ -0,0 +1,64 @@
+        uses: actions/cache/save@v3
+        with:
+          path: |
+            build/libs
+          key: ${{ steps.commit.outputs.short }}
+  release:
+    needs:
+      - build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)